tests/test_internals/test_interfaces.py

```diff
@@ -24,16 +24,17 @@
 from neuroconv.datainterfaces.ecephys.basesortingextractorinterface import BaseSortingExtractorInterface
 
 
+@pytest.mark.skipif(
+    platform != "darwin" or version.parse(python_version()) >= version.parse("3.8"),
```

Comment:
I somehow find the negative  of this more readable `platform == 'drawin' and version < 3.8`. Just a comment.

Reply:
I had it that way initially but found it less readable to have a big `not` in front of everything casted in parentheses. With this logic you can think of the conditions one at a time.

Not on a `darwin` platform? Skip.
Using Python version 3.8 or above? Skip.

No need to think about joint conditions this way

Reply:
I see, I was confused. I agree about it being more readable without distributing negation in your mind (the big negation in front of everything)

My confusion was that I read the reason
`reason="Not testing on MacOSX with Python<3.8!"`
And I thought that you wanted to skip the test if:
you are in the Darwin platform and the version of python was strictly before 3.8

That's why I wrote:
skip if:
`platform == 'drawin' and version < 3.8`

Am I mistaken that you are doing the opposite? Is the reason written wrong?

Reply:
The test, as seen below, only triggers when the user attempts to run the CED interface on a Mac device with Python Version 3.7.

Thus, running this test on anything other than that would fail because it wouldn't be triggered.

Thus, we skip this test if we are not testing on a Mac, or if we are using any Python version strictly above 3.7

Updated the `reason` to reflect this

Reply:
Thanks. It was the reason that threw me off.